BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -389,7 +364,7 @@ extension AttributedLabel {
                 }
 
                 assert(
-                    alignments.count == 1,
```

Comment:
This was causing a crash in the demo app. 
Someone went through the trouble of writing the nil case below, it'd be a shame never to use it.